styles/mixins/menu-slide.css

```diff
@@ -102,4 +102,38 @@ title: Mobile Menu
       }
     }
   }
+  &.mobile-only {
+    & .menu-btn {
+      & + label {
+        display: none;
+
+        @media (--tablet) {
+          display: block;
+        }
+      }
+      &:checked {
+        & ~ .menu-container {
+          right: 60%;
+          overflow-y: auto;
+        }
+      }
+      @media (--tablet) {
+        & ~ .menu-container {
+          position: fixed;
```

Comment:
I tired, but couldn't figure out how to.  Don't think we have extends options in PostCSS, and I need this nested underneath the parent selector.